### ISSUE TITLE
Create an "always safe / always non-sensitive" version of Openverse

### ISSUE THREAD

**Issue by zackkrida** [WordPress/openverse#4920]
## Problem

It seems likely that educational facilities using Openverse may want to restrict access to sensitive content on the platform. We've received feedback in the past relating to sensitive content from folks identifying as education-affiliated:

- Students saying they saw sensitive results while required to use Openverse for school
- Educators who have said they _would_ use Openverse, but can't due to sensitive content concerns 

## Description

Create a version of Openverse where the sensitive content filter is "locked" and cannot be modified. This could be handled in a number of ways, that should be determined by doing some research into how institutional firewalls commonly work. Some ideas though could be derived by how Google handles this with functionality called ["Lock SafeSearch for accounts, devices & networks you manage"](https://support.google.com/websearch/answer/186669?hl=en).

The tl;dr there is that they have network admins set a DNS CNAME record for forcesafesearch.google.com to www.google.com.

**Comment by obulat** [comment]
> One potential issue is the "readiness" of Openverse to introduce this functionality. Are our results sufficiently "safe" to make this assurance to users? Is there a safety threshold we could or should measure (some threshold of sensitive content reports, or something) we should determine first?

Maybe this version should use the non-user-generated media at first, as we do for some integrations?

**Comment by zackkrida** [comment]
@obulat that's actually a phenomenal idea that would help get this "up and running" quite quickly! Cool :sunglasses:
